File: C# Version/MenuProject/ChildForms/_ChildForm.cs
 
using System.Windows.Forms;

namespace MenuProject.ChildForms
{
    public partial class _ChildForm : Form
    {
        public _ChildForm()
        {
            InitializeComponent(); 
        }

        protected virtual void _childForm_Load(object sender, System.EventArgs e)
        {
            //Default settings for child forms
            this.ControlBox = false; 
            this.WindowState = FormWindowState.Maximized;
            //this.FormBorderStyle  = FormBorderStyle.None;
        }
    }
}

File: VB Version/VB MenuProject/ChildForms/_childForm.vb

Public Class _childForm
    Inherits Form

    Private Sub childForm_Load(ByVal sender As Object, _
        ByVal e As System.EventArgs) Handles MyBase.Load

        'Default settings for child forms
        Me.ControlBox = False
        Me.WindowState = FormWindowState.Maximized

    End Sub

End Class

